sys/shell/commands/sc_icmpv6_echo.c

```diff
@@ -38,9 +38,10 @@ static char ipv6_str[IPV6_ADDR_MAX_STR_LEN];
 
 static void usage(char **argv)
 {
-    printf("%s [<count>] <ipv6 addr> [<payload_len>] [<delay in ms>] [<stats interval>]\n", argv[0]);
+    printf("%s [<count>] <ipv6 addr[%%<device>]> [<interface>] [<payload_len>] [<delay in ms>] [<stats interval>]\n", argv[0]);
```

Comment:
uh, I had an extra interface parameter before I decided to go the split route. fixed (and now it's "interface" everywhere)